File: admin/lib/ui/screen/missing/model/GenderType.dart
enum GenderType {
  MALE,
  FEMALE
}

extension GenderTypeExtension on GenderType {
  String get rawValue {
    switch (this) {
      case GenderType.MALE:
        return "male";
      case GenderType.FEMALE:
        return "female";
    }
  }
}

File: iOS/PorPorCure/PorPorCure/Types/GenderType.swift
//
//  GenderType.swift
//  PorPorCure
//
//  Created by Jupond on 12/23/24.
//

import Foundation

enum GenderType: String {
    case MALE = "MALE"
    case FEMALE = "FEMALE"
    
    // 서버 값 변환 메서드 추가
       static func fromServerValue(_ value: String) -> GenderType? {
           switch value.uppercased() {
           case "MALE":
               return .MALE
           case "FEMALE":
               return .FEMALE
           default:
               return .MALE
           }
       }
}
